.github/workflows/build.yml

```diff
@@ -150,9 +150,9 @@ jobs:
     uses: howsoai/.github/.github/workflows/pytest.yml@main
     secrets: inherit
     with:
-      platform: 'macos-latest-large'
+      platform: 'macos-latest'
       platform-pretty: 'MacOS'
-      amalgam-plat-arch: 'darwin-arm64'
+      amalgam-plat-arch: 'darwin-amd64'
       python-version: '3.11'
       config-fp: './config/latest-mt-debug-howso.yml'
```

Comment:
Using this config will most likely fail sometimes since those runners usually don't have advanced intrinsics. They need a postfix of "-mt-noavx" to work right.